### PR TITLE
Update unions page with more recommendations

### LINKED ISSUE #2960: Issue on page /ethical-research/activism/activism-unionisation.html
Add unions of scientific and academic workers from Latin America: 

Argentina: 
APINTA (Asociación del Personal del Instituto Nacional de Tecnología Agropecuaria, Staff Association of the National Institute of Agricultural Technology) is the entity that groups together all workers, who have an employment relationship with INTA, as well as its retirees and pensioners. 

CONADU: Federación Nacional de Docentes Universitarios (National University Teachers Federation). Main objectives are the defense of:
Public, Free, Democratic, Popular, Feminist and Latin American University; salary, working conditions, training and teaching careers;     access, permanence and graduation of students in Higher Education; research and the production of knowledge, science and technology; the linkage of the university with the State and society to promote national and regional development.

### PR DESCRIPTION
**Pull request by Arielle-Bennett**
### Summary

Fixes #2960 

Adds @yabellini recommendations for unions in Lat Am + UTAW info + some formatting updates

### List of changes proposed in this PR (pull-request)

* Argentinian unions added
* UTAW added for UK
* Updated some formatting 

### What should a reviewer concentrate their feedback on?

- [ ] Everything looks ok?
- [ ] Do you have more suggestions for unions to add? 

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file.
- [x] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file: @yabellini for content

### CODE REVIEW

File: book/website/ethical-research/activism/activism-unionisation.md [hunk 1]
```diff
@@ -166,50 +166,56 @@ Other organisations that offer training and support on how to campaign effective
...
 
 #### United Kingdom
-* **Unite**, United Kingdom, the largest professional services union with over 1 million members [website](https://unitetheunion.org/)
-* **University & Colleges Union (UCU)**, United Kingdom, covers researchers, academic staff and higher grades of professional services staff in Higher Education & Further Education institutions [website](https://www.ucu.org.uk/)
-* **Prospect**, United Kingdom, union covering scientists & engineers [website](https://members.prospect.org.uk/our-industries/science/index)
-* **Public and Commercial Services Union (PCS)**, United Kingdom, civil service union [website](https://www.pcs.org.uk/about-pcs)
-* United Voices of the World (UVW), Charity Sector Workers Branch [website](https://www.uvwunion.org.uk/en/sectors/charity-sector/)
-* Independent Workers’ Union of Great Britain (IWGB), Charity Sector Workers Branch [website](https://iwgb.org.uk/page/iwgb-charity-workers-branch/)
+* **United Tech and Allied Workers (UTAW)**, United Kingdom, a branch of the Communication Workers Union, covering all workers at tech and data science companies, regardless of pay grade or relationship.
```
Comment (eirini-zormpa) [inline book/website/ethical-research/activism/activism-unionisation.md:172]: Could you explain what you mean by "relationship" here? Like, employment status at the company (full-time, part-time etc.)?

File: book/website/ethical-research/activism/activism-unionisation.md [hunk 2]
```diff
@@ -166,50 +166,56 @@ Other organisations that offer training and support on how to campaign effective
...
+* **Prospect**, United Kingdom, union covering scientists & engineers ([website](https://members.prospect.org.uk/our-industries/science/index))
+* **Public and Commercial Services Union (PCS)**, United Kingdom, civil service union ([website](https://www.pcs.org.uk/about-pcs))
+* **United Voices of the World (UVW), Charity Sector Workers Branch** ([website](https://www.uvwunion.org.uk/en/sectors/charity-sector/))
+* **Independent Workers’ Union of Great Britain (IWGB), Charity Sector Workers Branch** ([website](https://iwgb.org.uk/page/iwgb-charity-workers-branch/))
     * IWGB also has a number of branches covering cleaners, receptionists and game industry workers
 
 #### Ireland
-* **Medical Laboratory Scientists Association (MLSA)**, Ireland, is the sole trade union representing over 1600 Medical Scientists in Ireland [website](https://mlsa.ie/who-we-are)
+* **Medical Laboratory Scientists Association (MLSA)**, Ireland, is the sole trade union representing over 1600 Medical Scientists in Ireland ([website](https://mlsa.ie/who-we-are))
```
Comment (eirini-zormpa) [inline book/website/ethical-research/activism/activism-unionisation.md:183]:
```suggestion
* **Medical Laboratory Scientists Association (MLSA), Ireland** is the sole trade union representing over 1600 Medical Scientists in Ireland ([website](https://mlsa.ie/who-we-are))
```

Comment (eirini-zormpa) [reply book/website/ethical-research/activism/activism-unionisation.md:183]: I assume this is the Ireland chapter of a bigger organisation?

File: book/website/ethical-research/activism/activism-unionisation.md [hunk 3]
```diff
@@ -166,50 +166,56 @@ Other organisations that offer training and support on how to campaign effective
...
+* **Unión General de Trabajadores (UGT)** is a major Spanish trade union, historically affiliated with the Spanish Socialist Workers' Party (PSOE) ([website](https://www.ugt.es/))
+* **Confederación Sindical de Comisiones Obreras (CCOO)** is the second largest union in Spain and is the most successful one in labor elections ([website](https://www.ccoo.es/))
 * **Federacion de Jovenes Investigadores (FJI-Precarios)** is focused on improving the labour conditions of young researchers, defending the dignity of the research profession, and overall, improving the situation of science in Spain
-[website](http://precarios.org/Qui%C3%A9nes+somos)
+([website](http://precarios.org/Qui%C3%A9nes+somos))
 
+#### Argentina
+* **APINTA (Asociación del Personal del Instituto Nacional de Tecnología Agropecuaria, Staff Association of the National Institute of Agricultural Technology)** is the entity that groups together all workers, who have an employment relationship with INTA, as well as its retirees and pensioners ([website](https://www.apinta.org/))
```
Comment (eirini-zormpa) [inline book/website/ethical-research/activism/activism-unionisation.md:205]:
```suggestion
* **APINTA (Asociación del Personal del Instituto Nacional de Tecnología Agropecuaria, Staff Association of the National Institute of Agricultural Technology)** is the entity that groups together all workers who have an employment relationship with INTA, as well as its retirees and pensioners ([website](https://www.apinta.org/))
```

**Review by Davsarper — APPROVED** [review]
Inclussion of content is pertinent and edits are consistent.
Not including further unions at this point although will make a check on Spanish and potentially platform workers unions/groups in UK

**Review by eirini-zormpa — APPROVED** [review]

File: book/website/ethical-research/activism/activism-unionisation.md (hunk 1 above)
Comment (Arielle-Bennett) [reply book/website/ethical-research/activism/activism-unionisation.md:172]: Yes employment status is what I meant, will update 
```suggestion
* **United Tech and Allied Workers (UTAW)**, United Kingdom, a branch of the Communication Workers Union, covering all workers at tech and data science companies, regardless of pay grade or employment status (for example, part-time, full-time, or contractors). 
```

**Comment by Arielle-Bennett** [comment]
Latest commit is failing Lorum Ipsum test for no reason? Text added shouldn't have triggered that!

**Pull request merged**